packages/@jsii/node-check/lib/constants.ts

```diff
@@ -0,0 +1,57 @@
+export const enum SupportLevel {
+  /**
+   * Soft-deprecated, removed as of releases made starting at DEADLINE. This
+   * will output a yellow warning to the console, until the DEADLINE has been
+   * reached, at which point it'll be a SCARY RED WARNING.
+   */
+  DEPRECATED = 'end-of-life',
+
+  /**
+   * Supported releases, nothing to see. No message will be output.
+   */
+  SUPPORTED = 'supported',
+  /**
+   * EOL releases are unsupported, so this will emit a SCARY RED WARNING.
+   */
+  END_OF_LIFE = 'unsupported',
```

Comment:
Any reason for naming enum choices different from their string values? I feel like this could cause confusion perhaps

Reply:
Oh oops! This is the result of me being indecisive about naming... 🤓